src/component/MenuButton/MenuButton.js

```diff
@@ -0,0 +1,17 @@
+import './MenuButton.css';
+import hoverButton from '../../assets/hover-button.png';
+
+function MenuButton() {
+
+     return (
+        <div className='movie-tile_hover-button'>
+            <img src={hoverButton} alt='Hover Button' />
+            <div className="movie-tile_menu-content">
+                <a href="#" className='movie-tile_menu-link'>Edit</a>
```

Comment:
String literals should be moved to constant files.

Reply:
Fixed as part of 5ea7447